app/javascript/components/pages/EventShow.js

```diff
@@ -115,23 +118,15 @@ class EventShow extends Component {
           <>
             <h3>{this.state.drink.strDrink}</h3>
             <img src={this.state.drink.strDrinkThumb} alt='picture of drink' />
-            <div style={{display:'inline'}}>
+            <div style={{ display: 'inline' }}>
```

Comment:
it's been to be consistent about where styles live. If this file has a scss file, there shouldn't be any inline styling unless there's a veeeeeeeeery good reason

Reply:
this was just a baby step when we were scaffolding the api to see them go side by side but it didnt work and we just left in there.  we are working on this page next and well address this